code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -302,17 +302,17 @@
 /datum/game_mode/proc/auto_declare_completion_abduction()
 	var/text = ""
 	if(abductors.len)
-		text += "<br><span class='big'><b>The abductors were:</b></span>"
+		text += "<br><span class='big'><b>The abductors were:</b></span><br>"
 		for(var/datum/mind/abductor_mind in abductors)
 			text += printplayer(abductor_mind)
```

Comment:
We may need an additional new line here. If you look at the screenshot of the issue, there's no whitespace between player status and objective printout.